### PR TITLE
C2 mirna yacout c4 f4 record approval of a keyword

### PR DESCRIPTION
**Pull request by Mirna-Yacout**
#90

@AmrARaouf @Omar-Hossam @karimngr @Fattouh92 @m0hamed please review

work in Keyword.rb (disapprove_keyword method)
test: approve_keyword_spec.rb

### CODE REVIEW

**Comment by Mirna-Yacout** [comment]
test is renamed to disapprove_keyword_spec.rb

**Comment by omar-mohyeldin** [comment]
@Mirna-Yacout code reviewed

**Comment by abulazm22** [comment]
1) Please put title
2) change id with keyword_id
3) Keyword.find(keyword_id) should return keyword and then keyword.save

**Comment by Mirna-Yacout** [comment]
1- title is not necessary
2- id is written id not keyword_id for saving space and in all cases it's a variable so i could name it x
3- keyword.find actually returns a keyword and then keyword.save i cant actually find what u mean to change here

**Comment by abulazm22** [comment]
Okay, verified

File: arability/app/models/keyword.rb [hunk 1]
```diff
@@ -113,26 +113,28 @@ def self.find_by_name(name)
     return keyword
   end
 
-  class << self
   # Author:
   #  Mirna Yacout
   # Description:
-  #  This method is to record the aproval of the admin to a certain keyword in the database
+  #  This method is to record the disapproval of the admin to a certain keyword in the database
```
Comment (AmrARaouf) [inline arability/app/models/keyword.rb]: please check my comment on c4 f5 pull request and follow the convention here too

**Comment by Mirna-Yacout** [comment]
@AmrARaouf done

File: arability/app/models/keyword.rb [hunk 2]
```diff
@@ -113,26 +113,28 @@ def self.find_by_name(name)
...
-  #  or if the approval failed to be saved in the database 
-    def approve_keyword(keyword_id)
-      if Keyword.exists?(id: keyword_id)
-        keyword = Keyword.find(keyword_id)
-        keyword.approved = true
-        return keyword.save
-      end
-      return false
+  #   returns false if the keyword doesnot exist in the database
+  #      or if the disapproval failed to be saved in the database
```
Comment (AmrARaouf) [inline arability/app/models/keyword.rb]: no i meant that this line should be aligned with the one above it not offseted by 3 more spaces

**Comment by Mirna-Yacout** [comment]
@AmrARaouf please review

**Comment by AmrARaouf** [comment]
doc and testing reviewed

**Pull request merged**